FuelLabs/fuel-indexer
# chore: break out tests by db

### Description
- Self explanatory

### Testing steps
- [ ] Tests were added so CI should pass
  - Refresh DBs `bash scripts/utils/refresh_test_db.bash postgres` and `bash scripts/utils/refresh_test_db.bash sqlite`
  - Start test components `bash scripts/utils/start_test_components.bash`
  - Start the indexer (postgres) `cargo run --bin fuel-indexer`
  - Run e2e tests `bash packages/fuel-indexer-tests/scripts/e2e.bash`
  - Should all pass ✅ 

### Changelog
- chore: break out tests by db
